ipywidgets/static/widgets/less/widgets.less

```diff
@@ -188,6 +188,10 @@
     /* Listbox */
     width         : @widget-width;
     margin-bottom : 0px;
+    /* Bootstrap does some weird stuff with the margins of select boxes.  The 
+    specifity of the media selector needs to be overwritten using important */
+    margin-left   : 0px !important; 
+    margin-right  : 0px !important;
```

Comment:
@SylvainCorlay this may look alarming, but it's actually getting applied to the inner element, not the element that gets margin-ed by the traitlet, which is the outer element.